pycbc/distributions/sky_location.py

```diff
@@ -75,23 +76,15 @@ class FisherSky:
         Spread of the distribution. For the precise interpretation, see Eq 8
         of `Briggs et al 1999 ApJS 122 503`_. This should be smaller than
         about 20 deg for the approximation to be valid.
-    angle_unit: str
-        Unit for the angle parameters: either "deg" or "rad".
     """
 
     name = 'fisher_sky'
     _params = ['ra', 'dec']
 
     def __init__(self, **params):
-        if params['angle_unit'] not in ['deg', 'rad']:
-            raise ValueError("Only deg or rad is allowed as angle unit")
-        mean_ra = params['mean_ra']
-        mean_dec = params['mean_dec']
-        sigma = params['sigma']
-        if params['angle_unit'] == 'deg':
-            mean_ra = numpy.deg2rad(mean_ra)
-            mean_dec = numpy.deg2rad(mean_dec)
-            sigma = numpy.deg2rad(sigma)
+        mean_ra = angle_as_radians(params['mean_ra'])
+        mean_dec = angle_as_radians(params['mean_dec'])
+        sigma = angle_as_radians(params['sigma'])
```

Comment:
If you do the conversions here in the constructor, we need a couple more things:
1. Remove the float conversions in the `from_config()` method below, because they will not work if we have the suffixes, and the constructor now takes the strings directly.
2. Edit the docstring above to indicate that the parameters can now be float or string, and add the explanations of the suffixes like done [here](https://github.com/gwastro/pycbc/blob/master/bin/pycbc_multi_inspiral#L218-L219).

Reply:
I don't think we need to remove these conversions because [here](https://github.com/gwastro/pycbc/pull/4965/files#diff-7d65dce5b43e3ccad1a03ced076b28c76d415f7acbbc519cdb712dd823938897R566), the function is returning only the value and not the full string variable.

Reply:
`float('24deg')` will fail.